File: Latihan 1/bahasajava/Memory.java
//create derived class 2
class Memory extends Hardware{
	
	//private instance variables
	private String frequency;
	private String size; //memorySize
	private String cuda; //sportsCuda
	
	//constructors
	Memory(){
	}
	
	//public getter and setter for the private variable frequency
	public void setFrequency(String frequency){
		this.frequency = frequency;
	}
	//return variable frequency
	public String getFrequency(){
		return this.frequency;
	}
	
	//public getter and setter for the private variable size
	public void setSize(String size){
		this.size = size;
	}
	//return variable size
	public String getSize(){
		return this.size;
	}
	//public getter and setter for the private variable cuda
	public void setCuda(String cuda){
		this.cuda = cuda;
	}
	//return variable cuda
	public String getCuda(){
		return this.cuda;
	}
	
	
	
}
	

File: Latihan 2/bahasapython/Memory.py
from Hardware import Hardware

class Memory(Hardware):
#kelas yang digunakan untuk mengimplementasikan kelas memory

    def __init__(self):
    #konstruktor
        self.frequency = ""
        self.size = ""
        self.cuda = ""
        
    def setFrequency(self, frequency):
    #mengeset nilai atribut frequency
        self.frequency = frequency
        
    def getFrequency(self):
    #mengembalikan nilai atribut frequency
        return self.frequency
    
    def setSize(self, size):
    #mengeset nilai atribut size
        self.size = size
        
    def getSize(self):
    #mengembalikan nilai atribut size
        return self.size
        
    def setCuda(self, cuda):
    #mengeset nilai atribut cuda
        self.cuda = cuda
        
    def getCuda(self):
    #mengembalikan nilai atribut cuda
        return self.cuda
        
        